core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableValues.java

```diff
@@ -65,6 +65,19 @@ public static EnumerableValues create(RelOptCluster cluster,
     return new EnumerableValues(cluster, rowType, tuples, traitSet);
   }
 
+  /** Creates an EnumerableValues. */
+  public static EnumerableValues create(Values input) {
+    final RelOptCluster cluster = input.getCluster();
+    final ImmutableList<ImmutableList<RexLiteral>> tuples = input.getTuples();
+    final RelDataType rowType = input.getRowType();
+    final RelTraitSet traitSet =
+        input.getTraitSet()
+            .replace(EnumerableConvention.INSTANCE)
+            .replaceIf(RelDistributionTraitDef.INSTANCE,
+                () -> RelMdDistribution.values(rowType, tuples));
+    return new EnumerableValues(cluster, rowType, tuples, traitSet);
+  }
```

Comment:
Instead of doing this, can we just modify copy(traitset, inputs) method to just return `new EnumerableValues(getCluster(), rowType, tuples, traitSet);`?